cli/js/body.ts

```diff
@@ -131,6 +134,7 @@ export const BodyUsedError =
 
 export class Body implements domTypes.Body {
   protected _stream: domTypes.ReadableStream | null;
+  protected _bodyUsed = false;
```

Comment:
this uses a similar strategy to what was happening in fetch.ts before

https://github.com/denoland/deno/blob/master/cli/js/fetch.ts#L38

in that implementation we flip the switch when we issue our first read
https://github.com/denoland/deno/blob/master/cli/js/fetch.ts#L226

here, we flip the switch in the call to `arrayBuffer()`,